canonical-websites/docs.snapcraft.io
# Support multiple elements inside notifications

Add support for multiple paragraphs, or lists, or whatever, inside notifications.

Fixes #130

QA
--

`./run`

Go to http://0.0.0.0:8030/choosing-a-security-model/6847 and see that the first notification displays all the contents - 2 paragraphs and a list.

Go to http://0.0.0.0:8030/service-management/3965 to see that warning notifications still work.
